MauiStatus.md

```diff
@@ -1,5 +1,5 @@
 # MAUI Status
-This file tracks the progress of MAUI. Main branch has MSAL.NET code that is ported to MAUI. **It currently supports only iOS and Android platforms.** The branch has [two dev apps](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/tree/main/tests/devapps/MauiApps) used to for testing.   
+This file tracks the progress of MAUI. Main branch has MSAL.NET code that is ported to MAUI. **It currently supports only iOS and Android platforms.** The branch has [tthreewo dev apps](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/tree/main/tests/devapps/MauiApps) used to for testing.   

```

Comment:
This has a typo. Will fix with other inputs

Reply:
we should ideally drop the "number of apps", maintenance hazard